src/__tests__/AppLinks.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import {shallow} from 'enzyme';
+import React from 'react';
+import {AppLinks} from '../Header/AppLinks';
+
+describe('<AppLinks>', () => {
+  it('should add custom class name', () => {
```

Comment:
Please add test that will check that react render correct number of links

Reply:
Added